pom.xml

```diff
@@ -83,7 +83,7 @@
     <version.junit>4.13.1</version.junit>
     <version.maven-project>2.0.6</version.maven-project>
     <version.mockito>2.21.0</version.mockito>
-    <version.sonar>7.9</version.sonar>
+    <version.sonar>9.3.0.51899</version.sonar>
```

Comment:
We need tests to ensure that the ticket is fixed, something similar to [what was done for Python](https://github.com/SonarSource/sonar-python/pull/1094/files#diff-d2605292aaaeb585576f8ba373a78c5e0b75dc3793d1155ef57812668ae50a03R66-R77) would be fine.